hmrGC/image3d.py

```diff
@@ -335,7 +335,7 @@ def revert_trim_and_pad(self, xp=np):
         for i in range(self.signal.shape[-1]):
             tmp_arr = depad_array3d(self.signal[..., i], padsize)
             arr_new[..., i] = move2gpu(revert_trim_zeros(tmp_arr, slicing, shape), xp)
-        self.signal = move2cpu(arr_new, xp)

```

Comment:
I think it should be _move2cpu(arr_new, xp)_ here, but line 439 should be corrected to _arr = move2gpu(arr, xp)_